other/materials_designer/create_point_defect.ipynb

```diff
@@ -10,17 +10,10 @@
     "<h2 style=\"color:green\">Usage</h2>\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    for defect in DEFECT_CONFIGS:
</pre><p>for config in DEFECT_CONFIGS</p><p> </p><p>The if/else statement should go to PointDefectConfiguration constructor overloaded <code>from_dict</code> or something like that </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>